### PR TITLE
Added some songs JSON

### PR DESCRIPTION
**Pull request by olyanemyrovska**
Added JSON file

### CODE REVIEW

File: Lesson_2_JSON_YAML_XML/SomeSongs.json.txt
Comment (rodikno) [inline Lesson_2_JSON_YAML_XML/SomeSongs.json.txt:1]: The file has `.txt` extension (which might be added automatically), for specific representations we'd like to keep their "own" extensions like `.json`, `.yaml`, `.xml` so they can be better identified by humans and non-humans (aka programs which are processing these)

Comment (rodikno) [reply Lesson_2_JSON_YAML_XML/SomeSongs.json.txt:1]: As an additional plus - if the file has it's own extension -> tools like VSCode recognize the format and will offer syntax highlighting for you

File: Lesson_2_JSON_YAML_XML/SomeSongs.json.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "top_songs": [
+    {
+      "title": "Bohemian Rhapsody",
+      "duration_minutes": 5.55,
+      "release_year": 1975,
+      "album": "A Night at the Opera",
+      "is_single": true
+    },
+    {
+      "title": "Another One Bites the Dust",
```
Comment (rodikno) [inline Lesson_2_JSON_YAML_XML/SomeSongs.json.txt]: That's my favorite one :)

**Review by rodikno — APPROVED** [review]
Good job, approved (please take a look at the comments)

File: Lesson_2_JSON_YAML_XML/SomeSongs.json.txt
Comment (olyanemyrovska) [reply Lesson_2_JSON_YAML_XML/SomeSongs.json.txt:1]: Thanks for noticing! Downloaded VS code (finally)

**Pull request merged**